Field.php

```diff
@@ -161,18 +168,17 @@ private function nextGeneration()
      * Start-Game function
      * Prints the field, calculates the next Generation and prints the new field for "maxSteps"-times.
      * Also, if a field is printed double and the field does'nt change anymore, the program will stop automatically.
+     * @param $_output BaseOutput
      */
-    public function start()
+    public function start(&$_output)
```

Comment:
There is no reason to pass arguments by reference anymore. Objects are always passed by reference. So please remove the `&` before $_output.